tests/src/bin/gdb_c_test.rs

```diff
@@ -96,5 +96,5 @@ fn main() {
     }
 
     // Run gdb!
-    gdb.spawn().unwrap().wait().unwrap();
+    gdb.spawn().expect("failed to spawn gdb").wait().unwrap();
```

Comment:
I wasted about 10 minutes fiddling with variations of the test path, because I thought the missing file was the test file, not gdb. If it had told me right away, then I'd have known what was up. And that's me.. I wrote the tool and I still got confused.

(arguably this is a Rust problem, as the unwrap has a poor message IMHO -- it should say what path wasn't found)

18 bytes in a part of the system that isn't sensitive to memory or binary size... I wouldn't worry.

Reply:
OK.